The depmod program from kmod>=30 looks for modules.builtin.modinfo and throws a
warning when it isn't found. Newer versions of dracut pull in this file as well
as modules.builtin.alias, so let's pull this in here and avoid the warning.

--- a/dracut-init.sh
+++ b/dracut-init.sh
@@ -982,7 +982,7 @@
 }
 
 dracut_kernel_post() {
-    for _f in modules.builtin.bin modules.builtin modules.order; do
+    for _f in modules.builtin.{bin,alias,modinfo} modules.builtin modules.order; do
         [[ -e $srcmods/$_f ]] && inst_simple "$srcmods/$_f" "/lib/modules/$kernel/$_f"
     done
 
